src/StringTagField.php

```diff
@@ -412,4 +412,11 @@ public function setCanCreate($canCreate)
 
         return $this;
     }
+
+    public function performReadonlyTransformation()
+    {
+        $field = parent::performReadonlyTransformation();
+        $field->setValue(implode(', ', $this->Value()));
+        return $field;
+    }
```

Comment:
This PR originally used a `ReadOnlyField` - but this respects the parent classes and only makes the change that is necessary - setting the value as a string.

Reply:
Note that this is preferred over https://github.com/silverstripe/silverstripe-tagfield/pull/156 because it's _way_ simpler, and it doesn't add an entirely new class (which means both a smaller maintenance surface and we can ship this fix in a patch release)